hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeSetQuotaRequest.java

```diff
@@ -135,11 +135,12 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
       } else {
         omVolumeArgs.setQuotaInBytes(omVolumeArgs.getQuotaInBytes());
       }
-      if (checkQuotaCountsValid(setVolumePropertyRequest.getQuotaInCounts())) {
-        omVolumeArgs.setQuotaInCounts(
-            setVolumePropertyRequest.getQuotaInCounts());
+      if (checkQuotaCountsValid(
```

Comment:
Do we also change checkQuotaCountsValid to checkQuotaNamespaceValid?

Reply:
Indeed! Done!